open_dread_rando/files/exefs_patches/debug_input.s

```diff
@@ -0,0 +1,22 @@
+stp     x29, x30, [sp, -128]!
```

Comment:
Some comments on this file would be helpful to follow what's going on. And a comment on the top to explain the file.